packages/ceramic-core/src/store/level-state-store.ts

```diff
@@ -0,0 +1,171 @@
+import CID from "cids"
+import Level from 'level-ts';
+
+import type Ipfs from 'ipfs'
+import Document, { AnchorStatus, DocState } from "../document"
+import StateStore from "./state-store"
+
+/**
+ * Level backed State Store
+ */
+export default class LevelStateStore implements StateStore {
```

Comment:
We should probably have unit tests for this class?

Reply:
of course! 👍